flight/module/Makefile

```diff
@@ -0,0 +1,29 @@
+TOOLS_DIR      = /home/mlyle/dRonin/tools
```

Comment:
shouldn't be hardcoded

Reply:
Thx for review  :)

Yah-- The build system is completely wrong (in description as WIP and the individual commit comments as "DONTCOMMIT")  There's a lot of little things to figure out about how to find includes.

The modules stuff has to reside out of tree, and ... shouldn't depend on so many include dirs, etc.  It should also be considerably less complicated than our overall build system.  So there's work to be done on how the build system works for them.